Views/signup.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pdui.netlify.app/pdui.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+    <link rel="stylesheet" href="..//Components/Css/signup.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <nav>
+        <a class="active" href="..//Views/index.html">MyStore</a>
+        <input class="search-bar" type="text" placeholder="Search">
+        <a href="..//Views/cart.html">Cart<span class="material-icons-outlined">
+            shopping_cart
+          </span>
+              </a>
+        <a href="..//Views/wishlist.html">WishList<span class="material-icons-outlined">
+       favorite_border
+        </span>
+          </a>
+        <a href="..//Views/login.html">Login</a>
```

Comment:
I believe only one `/` should be in the path of href link, please check once. Kn other anchor tags as well

Reply:
okkk i will check ..